textBasedAdventure.js

```diff
@@ -1,5 +1,157 @@
+} else if (storyNodeFive === "c") {
+    console.log("Sir. Ma'am. A gas bomb just went off. Bye.");
+} else {
+    console.log("Error. Start game over.");
+}
+
+let storyNodeSeven = readline.question("You're underground and you see 3 giant red buttons. You can't understand what they're saying because they're in different languages but you have to pick one. Which one are you going for?  [A] German  [B] Russian  [C] Italian")
+
+if (storyNodeSix === "a") {
+    console.log("Oh no... you thought you did something.. There were 10 hungry rottweilers up there. Hope you can throw hands.");
+} else if (storyNodeSix === "b") {
+    console.log("Oh you got good luck. Aoowww." + storyNodeSeven);
+} else {
+    console.log("Error. Start game over.");
+}
+
+let storyNodeEight = readline.question("You've been transported to a beautiful paradise. The Polynesian islands to be exact. Too bad the creepy people somehow found their way there too. You see a giant rock to hide behind to the left and some trees to your right. Which way are you heading?  [A] left, toward the rocks  [B] right, to the trees.")
+
+if (storyNodeSeven === "a") {
+    console.log("You got transported to 1765 Germany. Idk what was going on then but sorry, you didn't make it.");
+} else if (storyNodeSeven === "b") {
+    console.log("...not sure why this was your answer but no, you didn't make it either.");
+} else if (storyNodeSeven === "c") {
+    console.log("I see you're moving on up. Okrrrrr." + storyNodeEight);
+}
+
+let storyNodeNine = readline.question("Have you watched that movie The Matrix before?   [A] yes     [B] no")
+
+
+if (storyNodeEight === "a") {
+    console.log("A flying arrow literally came out of nowhere. Wow, wasn't expecting that. Sorry. Bye.");
+} else if (storyNodeEight === "b") {
+    console.log("You made it to those trees safe and sound. One more hill to climb." + storyNodeNine);
+} else {
+    console.log("Error. Start game over.");
+}
+
+let storyNodeTen = readline.question("[A] Red pill     [B] Blue pill")
+
+if (storyNodeNine === "a") {
+    console.log("So do you remeber how Neo had to pick between the red and blue pill? Cool. Pick one." + storyNodeTen);
+} else if (storyNodeNine === "b") {
+    console.log("You really made it this far only to lose because you didn't watch the movie. A damn shame. You're out of here.");
+} else {
+    console.log("Error. Start game over.");
+}
+
+if (storyNodeTen === "a") {
+    console.log("Wow.. all that for nothing. You transported right back to where you started like that one episode of Black Mirror where the girl was stuck in a loop.");
+} else if (storyNodeTen === "b") {
+    console.log("Congratulations! It was just your best friends this whole time trying to surprise you with an all expense paid vacation to Bali. Who knew? 😅");
+} else {
+    console.log("Error. Start game over.");
+}
+
+
+
+// This game didn't really have any need for a switch statment but.... example for a bonus round
```

Comment:
I appreciate the effort but it would have been better to use it as part of the game.